packages/leva/package.json

```diff
@@ -22,19 +22,20 @@
     "react-dom": ">=16.8.0"
   },
   "dependencies": {
-    "@radix-ui/react-portal": "^0.1.3",
-    "@radix-ui/react-tooltip": "0.1.6",
+    "@radix-ui/react-portal": "^1.0.1",
+    "@radix-ui/react-tooltip": "1.0.2",
```

Comment:
why using a fixed version rather than "^1.0.2" (with ^caret) here?

Reply:
I upgraded with pnpm upgrade cli. Not sure what happened there tbh. That's weird. I'll fix this.

Reply:
> I upgraded with pnpm upgrade cli. Not sure what happened there tbh. That's weird. I'll fix this.

maybe `pnpm update --latest @radix-ui/react-tooltip` honours the original absence of caret (or respects the original exact version)

maybe `pmpm add` it?

Reply:
Fixed!